src/docs/welcome/gettingStarted/DOC.md

```diff
@@ -8,23 +8,18 @@ Storybook is a development environment for UI components. It allows you to brows
 
 ### Addons
 
-Addons are essentially plugins for Storybook that allow a deeper configuration
+Addons are essentially plugins for Storybook that allow a deeper configuration.
 
-#### [ReadMe](https://github.com/tuchk4/storybook-readme)
+These are the addons that are currently supported in this storybook.
 
-Storybook Community Addon Readme gives the ability to import markdown docs in the main story and in the Addon Panel
+* [ReadMe](https://github.com/tuchk4/storybook-readme) -
+    Storybook Community Addon Readme gives the ability to import markdown docs in the main story and in the Addon Panel
 
-#### [Knobs](https://github.com/storybooks/storybook/tree/release/3.4/addons/knobs)
+* [Knobs](https://github.com/storybooks/storybook/tree/release/3.4/addons/knobs) -
+    Allows you to edit React props dynamically using the Storybook UI. You can also use Knobs as a dynamic variable inside stories in Storybook.
 
-Storybook Addon Knobs allow you to edit React props dynamically using the Storybook UI. You can also use Knobs as a dynamic variable inside stories in Storybook.
-
-#### [Chapters](https://github.com/Checkfront/react-storybook-addon-chapters)
-
-React Storybook Chapters addon allows showcasing of multiple components within a story by breaking it down into smaller categories (chapters) and subcategories (sections) for more organizational goodness.
-
-#### [Actions](https://github.com/storybooks/storybook/tree/release/3.4/addons/actions)
-
-Storybook Addon Actions can be used to display data received by event handlers in Storybook.
+* [Chapters](https://github.com/Checkfront/react-storybook-addon-chapters) -
+    Allows showcasing of multiple components within a story by breaking it down into smaller categories (chapters) and subcategories (sections) for more organizational goodness.
```

Comment:
Hard to understand. If you mean you can break down the components, this should be:
`[...] by breaking them down into smaller categories [...]`
If you mean you can break down the story, it might be clearer as:
`[...] by breaking the story down into smaller categories [...]`

Reply:
This was taken from their ReadMe, but I'll update.